tools/validators/instance_validator/validate/subscriber.py

```diff
@@ -81,6 +82,8 @@ def Listen(self, callback):
       except (futures.CancelledError, KeyboardInterrupt):
         future.cancel()
       except Exception as ex:  # pylint: disable=broad-except
-        print(f"PubSub subscription failed with error: {ex}")
+        print('[ERROR]\tPub/sub subscription failed with error: {ex}'
```

Comment:
This could be changed into an f-string literal pretty easily. Might be worth it so we don't disable pylint unnecessarily.